distribution/docker/Dockerfile

```diff
@@ -47,12 +47,13 @@ RUN ["/busybox/busybox", "--install", "/bin"]
 
 RUN addgroup -S -g 1000 druid \
  && adduser -S -u 1000 -D -H -h /opt/druid -s /bin/sh -g '' -G druid druid \
- && mkdir -p /opt/druid/var \
- && chown -R druid:druid /opt \
- && chmod 775 /opt/druid/var
+ && mkdir -p /opt
 
 COPY --chown=druid:druid --from=builder /opt /opt
 COPY distribution/docker/druid.sh /druid.sh
+RUN mkdir -p /opt/druid/var \
+ && chown -R druid:druid /opt/druid/var \
```

Comment:
no need for `-R` here

Reply:
Yes, there's no sub-dirs under /opt/druid/var, `-R` is unnecessary. Fixed